lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -191,12 +186,14 @@ graceful_restart_timers_stalepath_time:
 
 log_neighbor_changes:
   kind: boolean
-  #TODO: Equivalent for bgp is 'bgp log'
-  # Currently supported but broken in IOS XR
+  cli_ios_xr:
+    default_value: true
+    config_get_token_append: '/^bgp log neighbor changes disable$/'
+    config_set_append: ' <state> bgp log neighbor changes disable'
   cli_nexus:
+    default_value: false
```

Comment:
They are different properties so not sure why them having different defaults across two platforms is an inconsistency.

So nexus is easy: enforce-first-as is on by default and log neighbor changes is not:

xyz(config)# router bgp 44
xyz(config-router)# vrf blue

xyz(config-router-vrf)# sh run bgp all | i enforce
  enforce-first-as
xyz(config-router-vrf)# sh run bgp all | i log

XR is harder because we can't list the defaults, but both enforce-first-as and log-neighbor-changes have 'disable' keywords. So as far as I can tell, they are both on by default and are disabled by adding the 'disable' keyword (and the command gets nvgenned).

So in my yaml they both have enforce-first-as default:true, and log-neighbor-changes has the difference in behavior.

It also affects the testing, meaning for log-neighbor I can't share quite so much testing logic:

XR (Default: true - set to false: disable keyword is added
Nexus (Default: false) - set to false - log neighbor command is removed if it is configured

XR - set to true: removes whole command 
Nexus - set to true: adds the log neighbor command

Clear as mud?